index.html

```diff
@@ -19,7 +19,11 @@ <h1>Welcome to the student showcase!</h1>
     </div>
     <div class="content">
         <div>ONE</div>
-        <div>TWO</div>
+        <div <img src="media/me.jpg" alt="Selfie">Allan Caprola
```

Comment:
Make sure that this is valid HTML (check your starting `<div>`), and make sure that it's indented properly to distinguish parents from children.

Reply:
Sorry but don't know how to fix it. I understand what you're saying but I can't figure it out.